docs/EN/resources/FAQ.md

```diff
@@ -0,0 +1,197 @@
+
+To understand the logic behind UAM, have a read of Time Streets blog post [Understanding SMB and oref1](https://diyps.org/2017/04/30/introducing-oref1-and-super-microboluses-smb-and-what-it-means-compared-to-oref0-the-original-openaps-algorithm/).
+
+**How does 'Max UAM SMB Basal Minutes' limit SMBs?**
+<br>`Max UAM SMB Basal Minutes` limits the size of SMBs that Trio can deliver when it detects an unannounced meal (UAM). The amount of insulin a UAM SMB can deliver is related to the amount of scheduled basal insulin in your "Basal Profile" settings. You can configure this setting to make UAM more or less aggressive in correcting meal spikes.
+
+Example: John has SMBs and UAM turned on in the settings. The algorithm has detected unexpected carb absorption and would like to deliver an SMB in response. Johns's current basal rate is if you are receiving a 1 unit per hour. John has `Max UAM SMB Basal minutes` set to the default of 30. Therefore, a single SMB can deliver no more insulin than would be delivered by your basal in 30 minutes, i.e., 0.5 units. 
+
+**I've turned on SMBs, but every time I receive an SMB, Trio cuts back or zero-temps my basal. Is that normal?**
+<br>Yes. SMBs effectively borrows insulin from the future to front load insulin delivery to assist in avoiding glucose spikes. Long, low, or zero temporary basal rates are expected to be observed with SMBs activated. This allows the algorithm to balance out the peak insulin timing and avoid hypoglycemia secondary to the increased insulin on board. 
+
+**How do I make SMBs more responsive?**
+<br>SMBs are limited by your Max `Max SMB Basal Minutes` and `Max UAM SMB Basal Minutes`. The default for both of these settings is 30 minutes. This default is conservative for most users, and it is suggested that this value be increased while monitoring the effect gradually. Raising this value will allow for bigger SMBs, thus making the dosing stronger. You need to decide how much you want to limit (or not limit) your SMBs.
+
+**Why isn't Trio looping when CGM glucose is 40 mg/dl (2.2 mmol/l) or lower?**
+<br>Trio can’t determine the glucose value when under 40mg/dL (2.2 mmol/L). This is why the CGM apps from Dexcom and Libre are displaying “Low.” A “Low” reading is also sometimes due to a sensor error or a faulty sensor needing replacement. Trio can't safely make a glucose prediction without knowing the glucose value or if it’s rising or falling.
+
+**Why isn't Trio looping or delivering insulin when CGM glucose is 400 mg/dl (22 mmol/l) or higher?**
+<br>Trio can’t determine the glucose value when over 400mg/dL (22 mmol/L). This is why the CGM apps from Dexcom and Libre display “High.” A “High” reading is also sometimes due to a sensor error or a faulty sensor needing replacement. Trio can't safely make a glucose prediction without knowing the glucose value or if it’s rising or falling.
```

Comment:
To add a line break after a sentence, instead of using the `<br/>` HTML tag, you can **add two spaces immediately after the last word of the sentence**. 
That is, use
> ```markdown
> [...] or if it’s rising or falling.␠␠
> ```
> ℹ️ In the above example, I used `␠` to **visually** denote and materialize a space character (` `).

instead of:
> ```markdown
> [...] or if it’s rising or falling.
> <br>
> ```

Reply:
Thanks for the tip.